defaults/main.yml

```diff
@@ -23,3 +23,4 @@ samba_local_master: 'yes'
 samba_domain_master: 'yes'
 samba_preferred_master: 'yes'
 samba_mitigate_cve_2017_7494: true
+samba_python2: true
```

Comment:
To make this work on Fedora 33 I had to change this to `false`. Looking at the logic, I think this actually should default to false, as when the python2 packages are needed, this variable is set to the relevant package name(s).